flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutorResourceUtils.java

```diff
@@ -250,7 +251,7 @@ private static void warnOptionHasNoEffectIfSet(Configuration config, ConfigOptio
                     "The resource configuration option {} is set but it will have no effect for local execution, "
                             + "only the following options matter for the resource configuration: {}",
                     option,
-                    UNUSED_CONFIG_OPTIONS);
+                    CONFIG_OPTIONS);
```

Comment:
This should be `CONFIG_OPTIONS` as they are the ones that matter. The `UNUSED_CONFIG_OPTIONS` is not effective for local executor.